samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java

```diff
@@ -1237,4 +1246,176 @@ public String getExportStatisticsPdf() {
     private String exportReportUrl(String type, String format) {
         return ExportReportServlet.exportReportUrl(assessmentId, type, format);
     }
+
+  public String[] getTimeStats() {
+    boolean firstTimeInside = true;
+    int timeElapsedInSeconds = 0;
+    int timeMin = 0;
+    int timeAvg = 0;
+    int timeMax = 0;
+    int counter = 0;
+    GradingService gradingService = new GradingService();
+    if (this.getAllSubmissions().equals("3")) {
+      TotalScoresBean totalBean = (TotalScoresBean) ContextUtil.lookupBean("totalScores");
+      List allSubmissionsList = gradingService.getAllSubmissions(this.getPublishedId());
+      for (Object submission : allSubmissionsList) {
+        AssessmentGradingData assessmentGradingAux = (AssessmentGradingData) submission;
+        try {
+          timeElapsedInSeconds = ((int) assessmentGradingAux.getSubmittedDate().getTime()) / 1000 - ((int) assessmentGradingAux.getAttemptDate().getTime()) / 1000;
+        } catch (Exception ex) {
+          timeElapsedInSeconds = 0;
```

Comment:
Add 

log.error("Cannot resolve the submittedDate or the attemptDate, so it's a unanswered question, setting time as 0");

like on https://github.com/sakaiproject/sakai/pull/12334/files#diff-6efbfc756f2a2c4ad26c1e1fc5fdc24c2114df9618f98edf74dba074eaefa736R1308?